values.yaml

```diff
@@ -37,23 +47,32 @@ octopus:
   # Azure AKS uses azurefile for this ReadWriteMany volumes.
   # Google uses Filestore for ReadWriteMany volumes: https://cloud.google.com/filestore/docs/accessing-fileshares
   # Amazon uses EFS for ReadWriteMany volumes: https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html
-  storageClassName: ""
+  storageClassName: "test"
+
   # When replica count is 1, this value defines the storage access type. When replica count is greater than 1, the storage more is always ReadWriteMany.
-  storageAccessMode: ReadWriteOnce
+  storageAccessModeOneReplica: ReadWriteOnce
+  storageAccessModeManyReplicas: ReadWriteMany
+
   # The size of the repository volume
   repositoryVolumeSize: 1Gi
+
   # The size of the artifact volume
   artifactVolumeSize: 1Gi
+
   # The size of the task log volume
   taskVolumeSize: 1Gi
+
   # The port to access the web interface
   webPort: 80
   # The port to access the polling tentacles on
+
   tentaclePort: 10943
```

Comment:
`repositoryVolumeSize`, `artifactVolumeSize`, `taskVolumeSize`, `tentaclePort`, and `webPort` are no longer used? I think these values still make sense as configurable options.